akvo/urls.py

```diff
@@ -89,6 +89,9 @@
     url(r'^activate/(?P<activation_key>\w+)/$',
         'akvo.rsr.views.account.activate', name='activate'),
 
+    url(r'^activate_invite/(?P<inviting_pk>\d+)/(?P<pk>\d+)/(?P<token_date>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z_\-]+)/$',
+        'akvo.rsr.views.account.invite_activate', name='invite_activate'),
```

Comment:
For now I've kept this the same (even added an employment.pk to the URL), since I'm not 100% sure the current invite process could work using the default activation link.